### DISCUSSION TITLE
Custom react-select filter for a column is not showing its dropdown in react-table

### DISCUSSION THREAD

**Discussion by niikkiin** [TanStack/table#3876] [Q&A]
Hi! I've been trying to solve this problem since last week and I could really use some help with this one. In this sample codesandbox, my custom react-select filter for a column is not showing it dropdown, it should be on top of the body but instead, it's being cutoff inside the thead. Thanks in advance!

**SANDBOX:** https://codesandbox.io/s/react-table-custom-filtering-multiple-values-filter-forked-uej3jo?file=/index.js
Another example: https://codesandbox.io/s/react-select-dropdown-not-showing-in-react-table-custom-filter-rnefzh

![image](https://user-images.githubusercontent.com/26428605/166399740-09b0e589-cd3a-4c2c-8216-2f620fd282b0.png)
![image](https://user-images.githubusercontent.com/26428605/166399882-e57caa6b-a6d2-4baa-b3a4-fd68794a6185.png)

**Comment by Mdakhib** [comment]
@niikkiin did you got any solution?

**Reply by niikkiin** [reply]
unfortunately, not yet. I haven't found a workaround for this.

**Comment by talatkuyuk** [comment]
It seems the problem belongs to the react-select, not to react-table. It may be simple CSS trick as well.
https://stackoverflow.com/questions/55830799/how-to-change-zindex-in-react-select-drowpdown

In any way, if you find any solution, waiting to see it..
